e2e_tests/test_workspace_creation.py

```diff
@@ -34,46 +34,4 @@ async def test_getting_templates(template_name, admin_token, verify) -> None:
         assert (response.status_code == status.HTTP_200_OK), f"GET Request for {template_name} creation failed"
 
 
-@pytest.mark.extended
-@pytest.mark.timeout(1800)
-async def test_create_devtestlabs_workspace(workspace_owner_token, admin_token, verify) -> None:
```

Comment:
Why are these ones gone? Did you intend to replace them with tests for `post_workspace` (as you import that method)?

Reply:
these workspace types don't exist any more, but the tests still hung around